README.md

```diff
@@ -10,15 +10,17 @@
 
 ## Disclaimer
 
-This repo is a [monorepo](https://en.wikipedia.org/wiki/Monorepo#:~:text=In%20version%20control%20systems%2C%20a,as%20a%20'shared%20codebase'.) consisting mainly of npm packages. We use [workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces) to managing the different npm packages etc. This may change in the future as we plan to include also packages, libraries etc. written in other languages in this repo to make the power of AskUI available to non-typescript/-javascript developers as well.
+This repo contains the AskUI SDK (ADK) written in TypeScript. Releases are done from the root repository. This may change in the future as we plan to include also packages, libraries etc. written in other languages in this repo to make the power of AskUI available to non-typescript/-javascript developers as well.
 
 ## Installation
-Run an `npm install` inside the root directory.
+Run an `npm install` inside the root directory if you want to create a new release.
```

Comment:
I don't think it is super clear for a user what is going on here.

1. I would explain the project structure first which explains why `npm install` is to be run twice at different hierarchy levels. 
2. Then, I would explain what is installed in root (I don't think this  about releasing or at least not only but rather committing etc.) vs. in the `packages/askui-nodejs`.